src/Pages/ContentListTable/components/SnapshotListModal/SnapshotListModal.tsx

```diff
@@ -143,7 +143,13 @@ export default function SnapshotListModal() {
   const fetchingOrLoading = isFetching || isLoading;
 
   const loadingOrZeroCount = fetchingOrLoading || !count;
+  const SnapshotListModalRoute = (snapUUID) => `/content/${uuid}/snapshots/${snapUUID}/packages`;
 
+  const onPackageCountClick = (snapUUID) => {
+    const SnapshotListModalRoute = `/content/${uuid}/snapshots/${snapUUID}/packages`;
+
+    navigate(`${rootPath}${SnapshotListModalRoute}`);
+  };
```

Comment:
You have two functions doing the same thing, and the routes aren't correct on either one. 
The[ route you created](https://github.com/content-services/content-sources-frontend/pull/207/files#diff-21c0871aec2cc959996fa3600d513fccb3575d5d72dd892b0102df47b62058acR45) below does not match.

Choose one of the above functions/methods to navigate.